src/app/dashboard/Missions/MissionFundedStatusRow.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { Mission } from "../../model";
+import AttachMoneyIcon from "@material-ui/icons/AttachMoney";
+import { Box, Container, Grid, Paper } from "@material-ui/core";
+
+const Row = ({ children, Icon }) => {
+  if (!children) return null;
```

Comment:
Row seems to be a general name for this component, but with a pretty specific use case here.

We may not need to even have a separate component for `Row` because of the nature of the parent it's living in.

In this case, i would just put the JSX from `Row` inside your return for `MissionSFundedStatusRow` component.